toolchain/check/import_ref.cpp

```diff
@@ -2717,6 +2742,9 @@ static auto TryResolveInstCanonical(ImportRefResolver& resolver,
     case CARBON_KIND(SemIR::UnboundElementType inst): {
       return TryResolveTypedInst(resolver, inst);
     }
+    case CARBON_KIND(SemIR::Vtable inst): {
+      return TryResolveTypedInst(resolver, inst, inst_id);
```

Comment:
Why pass `inst_id` if it isn't going to be used?

Reply:
Ah, thanks for the catch - removed the unused parameter in #4832